dids/src/main/kotlin/web5/sdk/dids/methods/dht/DidDht.kt

```diff
@@ -77,14 +79,13 @@ private class DidDhtApiImpl(configuration: DidDhtConfiguration) : DidDhtApi(conf
 
 /**
  * Specifies options for creating a new "did:dht" Decentralized Identifier (DID).
- * @property verificationMethods A list of [JWK]s to add to the DID Document mapped to their purposes
- * as verification methods.
- * @property services A list of [Service]s to add to the DID Document.
+ * @property verificationMethodsToAdd List of specs that will be added to the DID ION document.
+ * @property servicesToAdd A list of [Service]s to add to the DID Document.
  * @property publish Whether to publish the DID Document to the DHT after creation.
  */
 public class CreateDidDhtOptions(
-  public val verificationMethods: Iterable<Pair<JWK, Array<PublicKeyPurpose>>>? = null,
-  public val services: Iterable<Service>? = null,
+  public val verificationMethodsToAdd: Iterable<VerificationMethodSpec> = emptyList(),
+  public val servicesToAdd: Iterable<Service> = emptyList(),
```

Comment:
`toAdd` seems redundant here given that these are _create_ options.

Reply:
i can create a separate issue for changing `DidIon`. `DidDht` was already just `services` and `verificationMethods`. We should keep it that way. no point in changing code to match what's going to be immediately changed

Reply:
Ah, I see what you mean. Fixed in [c5b2eac](https://github.com/TBD54566975/web5-kt/pull/146/commits/c5b2eac1b5447a56f0600d340151f8a2790fdc38)

Reply:
And renamed to the existing didion to [b24b9fd](https://github.com/TBD54566975/web5-kt/pull/146/commits/b24b9fdcbdb63ef1a41465381165c1154291ad0f)